src/java/fr/paris/lutece/plugins/forms/service/search/FormsSearchIndexerDaemon.java

```diff
@@ -58,7 +58,8 @@ public class FormsSearchIndexerDaemon extends Daemon {
     @Override
     public void run() {
 
-        if( DATASTORE_DAEMON_MODE_FULL.equals(DatastoreService.getDataValue(DATASTORE_DAEMON_MODE,DATASTORE_DAEMON_MODE_INCREMENTAL)) )
+        if( !_formSearchIndexer.isIndexerInitialized( ) ||
```

Comment:
For huge databases, we should have the opportunity to disable this behaviour,
could you please add a property to activate this automatic reindexing feature ? 
(set default to TRUE)